### ISSUE TITLE
Is there a way to access the metadata for the file being uploaded?

### ISSUE THREAD

**Issue by tizol** [superflows-dev/react-upload-to-s3#1]
Is there a way to access the metadata for the file being uploaded, like file size and name? Also, when I capture a video's thumbnail and click preview of the thumbnail. The button "Close x"  of the modal to preview the thumbnail is not visible if the thumbnail is color white. 
<img width="935" alt="Screen Shot 2022-11-09 at 3 32 16 PM" src="https://user-images.githubusercontent.com/6495684/200944345-064e16c3-7a5d-4fee-92c9-40382c5acbae.png">

What about disabling the thumbnail feature if it's not needed?

**Comment by superflows-dev** [comment]
```
<UploadToS3 
    bucket="myuploads"
    awsRegion="awsRegion"
    awsKey="awsAccessKey"
    awsSecret="awsSecret"
    awsMediaConvertEndPoint="https://*********.mediaconvert.<awsRegion>.amazonaws.com"
    type="pdf"
    mediaConvertRole="mediaconvert_role"
    theme={theme}
    showNewUpload={false}
    onResult={(result) => {console.log('on Result', result);}} />
```

*result* argument of the onResult callback returns the meta information.

Please upgrade to version 1.0.13, Close button issue is fixed. 

Why disable the thumbnail feature? Any use-case that justifies disabling?

**Comment by superflows-dev** [comment]
> Is there a way to access the metadata for the file being uploaded, like file size and name? Also, when I capture a video's thumbnail and click preview of the thumbnail. The button "Close x" of the modal to preview the thumbnail is not visible if the thumbnail is color white. <img alt="Screen Shot 2022-11-09 at 3 32 16 PM" width="935" src="https://user-images.githubusercontent.com/6495684/200944345-064e16c3-7a5d-4fee-92c9-40382c5acbae.png">
> 
> What about disabling the thumbnail feature if it's not needed?

<UploadToS3 
    bucket="myuploads"
    awsRegion="awsRegion"
    awsKey="awsAccessKey"
    awsSecret="awsSecret"
    awsMediaConvertEndPoint="https://*********.mediaconvert.<awsRegion>.amazonaws.com"
    type="pdf"
    mediaConvertRole="mediaconvert_role"
    theme={theme}
    showNewUpload={false}
    onResult={(result) => {console.log('on Result', result);}} />
result argument of the onResult callback returns the meta information.

Please upgrade to version 1.0.13, Close button issue is fixed.

Why disable the thumbnail feature? Any use-case that justifies disabling?

**Comment by tizol** [comment]
This is what the result returns: 
`{ result: true, url: 'imgsteleport/video_1667945031664.mp4', thumbnail: 'imgsteleport/video_1667945038224.jpg' }`

It doesn't return the file name, file type, size, width, and height. For both, the main video file and thumbnail. 

Also, when I capture a video's thumbnail and it gets uploaded as: 

<img width="629" alt="Screen Shot 2022-11-10 at 11 56 40 AM" src="https://user-images.githubusercontent.com/6495684/201158463-5ac1de97-fedf-417f-8a5c-ed169ac8f27d.png">

**Comment by tizol** [comment]
I see you have them saved in state @ [react-upload-to-s3/blob/main/src/index.js#L74-L76](https://github.com/superflows-dev/react-upload-to-s3/blob/main/src/index.js#L74-L76)

```js
  const [srcThumbnail, setSrcThumbnail] = useState('')
  const [videoName, setVideoName] = useState(null);
  const [videoDuration, setVideoDuration] = useState(null);
  const [videoSize, setVideoSize] = useState(null);
  const [videoWidth, setVideoWidth] = useState(0);
  const [videoHeight, setVideoHeight] = useState(0);
  const [videoCurrentTime, setVideoCurrentTime] = useState(null);
  const [videoStartPosition, setVideoStartPosition] = useState(null);
  const [videoEndPosition, setVideoEndPosition] = useState(null);
  const [pdfName, setPdfName] = useState('')
  const [pdfSize, setPdfSize] = useState('')
  const [disableMarkStart, setDisableMarkStart] = useState(false);
  const [disableMarkEnd, setDisableMarkEnd] = useState(false);
```

**Comment by superflows-dev** [comment]
Please upgrade to 1.0.14

- Image not showing when thumbnail url used > fixed
- onResult not returning video meta data >  fixed (returning current parameters + processedSize, processedDuration, originalName, originalSize, originalDuration)

**Comment by tizol** [comment]
Thanks, @superflows-dev! 
- thumbnail image is working correctly 
- and metadata for the original file is returned 

I submitted a Pull Request https://github.com/superflows-dev/react-upload-to-s3/pull/2 to return originalVideoWidth originalVideoHeight. Let me know if you could merge those changes.

**Comment by superflows-dev** [comment]
@tizol merged and deployed to 1.0.15
